monitor/alerts/test.py

```diff
@@ -1,37 +1,29 @@
 import json
 import re
-from datetime import datetime
-from typing import List, Optional
+from typing import Optional
 
 from slack_sdk.models.blocks import SectionBlock
 
 from clients.slack.schema import SlackMessageSchema
 from monitor.alerts.alert import Alert
-from utils.json_utils import try_load_json, prettify_json_str_set
+from utils.json_utils import try_load_json
 from utils.log import get_logger
-from utils.time import convert_utc_time_to_local_time
 
 logger = get_logger(__name__)
 
 
 class TestAlert(Alert):
+    TABLE_NAME = 'alerts'
+
     def __init__(
             self,
             model_unique_id: str,
             test_unique_id: str,
-            status: str,
-            id: str,
-            elementary_database_and_schema: str,
-            subscribers: Optional[List[str]] = None,
-            slack_channel: Optional[str] = None,
             **kwargs
     ) -> None:
-        super().__init__(id, elementary_database_and_schema, subscribers, slack_channel)
+        super().__init__(**kwargs)
```

Comment:
Why are you now passing kwargs to all alerts? isn't it better to have explicit names? is it just because you repeat their declaration?

Reply:
I want each `Alert` class to extract the fields that are relevant to itself and pass the rest onwards to the parent.
Much cleaner, concise and I don't have to change the `__init__()` on all the alerts when I add a new parameter.
That's how `dataclasses` work by the way.